app/src/common/constants/widgetTypes.js

```diff
@@ -36,3 +36,27 @@ export const MOST_TIME_CONSUMING = 'mostTimeConsuming';
 export const MOST_POPULAR_PATTERNS = 'topPatternTemplates';
 export const COMPONENT_HEALTH_CHECK = 'componentHealthCheck';
 export const COMPONENT_HEALTH_CHECK_TABLE = 'componentHealthCheckTable';
+
+export const HUMAN_WIDGET_TYPES_MAP = {
```

Comment:
You can use default messages from `widgetTypesMessages` (`src/pages/inside/dashboardItemPage/modals/common/widgets.jsx`) instead of creating the new one map.